v2/mongodb-to-googlecloud/docs/MongoDbToBigQuery/mongodb-to-bigquery-image-spec-metadata.json

```diff
@@ -37,8 +37,20 @@
       "is_optional": false,
       "paramType": "TEXT",
       "regexes": [
-        "^(FLATTEN|NONE)$"
+        "^(FLATTEN|NONE|UDF)$"
```

Comment:
Also, why is userOption "is_optional" set to false here, but true in the mongodb-to-bigquery-cdc-image-spec-metadata.json? Can't all instances of this parameter be optional?

Reply:
IT should be true updating it to True